android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -661,7 +770,11 @@ public void selectItem(int sectionIndex, int itemIndex)
 				final ListItemProxy item = section.getListItemAt(itemIndex);
```

Comment:
If I set up a ListView to enable editing mode via a "longpress" event (which is how most Android app do it), you would normally want to check the item you long-pressed and you would do this via our `selectItem()` method (see my "longpress" event listener below). The issue with this is the 1st line in this Java `selectItem()` method calls `scrollToItem()`, which we should do if the item is offscreen... but... we don't want the ListView to suddenly scroll the ListView if the item is already onscreen, such as for our long press use-case.
_(Note that iOS will scroll to the selected item if offscreen, so, that part is okay.)_

Can you use the `getFirstVisibleItem()` and `getLastVisibleItem()` methods please and only call `scrollToItem()` if offscreen?

```javascript
function createSection(rows, suffix) {
	const items = [];
	for (let index = 1; index <= rows; index++) {
		items.push({ properties: { title: `Row #${index}` } });
	}
	const section = Ti.UI.createListSection({
		headerTitle: `Section ${suffix}`,
		items: items,
	});
	return section;
}

const window = Ti.UI.createWindow();
const listView = Ti.UI.createListView({
	sections: [
		createSection(10, 'A'),
		createSection(10, 'B'),
		createSection(10, 'C')
	],
	editing: false,
	allowsMultipleSelectionDuringEditing: true,
	allowsSelection: false,
	allowsSelectionDuringEditing: true,
	showSelectionCheck: true,
});
listView.addEventListener('longpress', (e) => { // Event not supported on iOS.
	if (e.section && !listView.editing) {
		listView.editing = true;
		listView.selectItem(e.sectionIndex, e.itemIndex);
	}
});
listView.addEventListener('itemsselected', (e) => {
	if (e.selectedItems.length <= 0) {
		listView.editing = false;
	}

	let text = '';
	for (const nextItem of e.selectedItems) {
		if (text.length > 0) {
			text += ',';
		}
		const section = nextItem.section;
		text += ` [${section.headerTitle}][${section.getItemAt(nextItem.itemIndex).properties.title}]`;
	}
	console.log(`itemsselected: ${text}`);
});
window.add(listView);
window.open();
```